apps/examples/elf/wifiapp/wifiapp.c

```diff
@@ -27,6 +27,20 @@
 #include <time.h>
 #include <sys/time.h>
 
+
+#ifdef CONFIG_EXAMPLES_MPU_TEST
+
+#ifdef CONFIG_ARCH_CHIP_IMXRT
+#define KERNEL_CODE_START_ADDR	0x60000000
+#define KERNEL_DATA_START_ADDR	0x20200000
+#define APP_START_ADDR		0x20250000
```

Comment:
If they are required, lets make configurations in Kconfig. I don't want to have hard-coded board-specific values in common codes.

Reply:
ok. fixed